packages/data-model/src/types.ts

```diff
@@ -120,7 +120,13 @@ export interface ApplicantDTO {
 	applicantPrimaryAffiliation?: string | null;
 	applicantInstitutionalEmail?: string | null;
 	applicantProfileUrl?: string | null;
+	applicantInstituteCountry?: string | null;
```

Comment:
Super minor but can you move this below `applicantPositionTitle` so all Institute address fields are grouped?